cw_4/src/Pages/product/Product.jsx

```diff
@@ -1,62 +0,0 @@
-
-import React from 'react'
-import axios from 'axios'
-import { useEffect } from 'react'
-import { useState } from 'react'
-
-const Product = () => {
-    const [loading,setLoading]=useState(true)
-const [data,setData]=useState([])
-
-useEffect(()=>{
-    setLoading(true)
-    axios.get("https://backend-cw-4.onrender.com/Products")
-    .then((res)=>setData(res.data))
-    setLoading(false)
```

Comment:
this setLoading should be inside the .then statement because the Axios call is asynchronous and here setLoading call will execute before the Axios call is finished